packages/cli/src/PublicApi/v1/openapi.yml

```diff
@@ -1,34 +1,30 @@
 ---
 openapi: 3.0.0
 info:
-  title: Public n8n API
+  title: n8n Public API
   description: n8n Public API
   termsOfService: https://n8n.io/legal/terms
   contact:
     email: hello@n8n.io
   license:
     name: Sustainable Use License
     url: https://github.com/n8n-io/n8n/blob/master/packages/cli/LICENSE.md
-  version: 1.0.0
+  version: v1
```

Comment:
Anything speaking against keeping the `major.minor.patch` versioning scheme here?